### PR TITLE
Set up CI/CD pipeline

### PR DESCRIPTION
**Pull request by kiron-mx**
Configured GitHub Actions for

- Code format check
- Code quality check
   - Sonarcloud code quality
   - OWASP vulnerability check
- Licence checker

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -2,12 +2,12 @@
...
 <project xmlns="http://maven.apache.org/POM/4.0.0"
-	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+		 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>com.siemens.pki</groupId>
 	<artifactId>LightweightCmpRa</artifactId>
-	<version>4.0.1</version>
+	<version>4.0.0</version>
```
Comment (ralienpp) [inline pom.xml:10]: The version number should be updated.

File: pom.xml [hunk 2]
```diff
@@ -141,7 +148,7 @@
 				<configuration>
 					<serverId>ossrh</serverId>
 					<nexusUrl>https://s01.oss.sonatype.org/</nexusUrl>
-					<autoReleaseAfterClose>true</autoReleaseAfterClose>
+					<autoReleaseAfterClose>false</autoReleaseAfterClose>
```
Comment (ralienpp) [inline pom.xml:151]: This thing will not be published on MavenCentral, are you sure we need this plugin?

**Review by ralienpp — CHANGES_REQUESTED** [review]

File: pom.xml [hunk 3]
```diff
@@ -150,19 +157,33 @@
...
+				</executions>
+			</plugin>
 		</plugins>
 	</build>
 	<dependencies>
 		<dependency>
 			<groupId>com.siemens.pki</groupId>
 			<artifactId>CmpRaComponent</artifactId>
-			<version>4.1.0</version>
+			<version>4.0.0</version>
```
Comment (ralienpp) [inline pom.xml:186]: This version can also be updated now, to be aligned with the latest cmpracomponent

File: pom.xml (hunk 1 above)
Comment (kiron-mx) [reply pom.xml]: Changed in code

File: pom.xml (hunk 2 above)
Comment (kiron-mx) [reply pom.xml]: Deleted dependency

File: pom.xml (hunk 3 above)
Comment (kiron-mx) [reply pom.xml]: Changed in code

**Review by ralienpp — APPROVED** [review]

**Pull request merged**